### PR TITLE
Add a bio for @musicfuel

### PR DESCRIPTION
**Pull request by musicfuel**
This pull request adds a bio for @musicfuel as requested in #18

### CODE REVIEW

File: docs/musicfuel.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#musicfuel
```
Comment (heightriders) [inline docs/musicfuel.md]: Consider capitalizing the first letter on the Title

File: docs/musicfuel.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+### Some basics:
+- **Name:** James Newell
+- **Hometown:** Sheboygan, WI
+- **Occupation:** Technical Architect with [SafeNet](http://www.safenetconsulting.com/)
```
Comment (heightriders) [inline docs/musicfuel.md:5]: Good job !!! you are hired

File: docs/musicfuel.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#musicfuel
+
+### Some basics:
```
Comment (githubteacher) [inline docs/musicfuel.md]: I like this line here, but I wonder if we may want to remove it for now since there aren't any other subsections yet.

**Comment by githubteacher** [comment]
I made an inline comment for something you may want to address, but after that this looks :+1:  to me.

File: docs/musicfuel.md (hunk 1 above)
Comment (musicfuel) [inline docs/musicfuel.md]: Done

File: docs/musicfuel.md (hunk 3 above)
Comment (musicfuel) [inline docs/musicfuel.md]: Good point. No need for a header at this time :+1:

**Pull request merged**